test/integration/image-component/typescript/test/index.test.js

```diff
@@ -27,6 +29,21 @@ describe('TypeScript Image Component', () => {
       expect(stderr).toMatch(/Failed to compile/)
       expect(stderr).toMatch(/is not assignable to type/)
       expect(code).toBe(1)
+      const envTypes = await fs.readFile(join(appDir, 'next-env.d.ts'), 'utf8')
+      expect(envTypes).toContain('image-types/global')
+    })
+
+    it('should remove global image types when disabled', async () => {
+      const content = await fs.readFile(nextConfig, 'utf8')
+      await fs.writeFile(
+        nextConfig,
+        content.replace('// disableStaticImages', 'disableStaticImages')
+      )
+      const { code } = await nextBuild(appDir, [], { stderr: true })
+      expect(code).toBe(1)
```

Comment:
How about adding an assertion for stderr here?

Reply:
Added a similar check from the above the test